sdk/core/azure-core/test/ut/service_tracing_test.cpp

```diff
@@ -20,13 +20,15 @@ TEST(TracingContextFactory, ServiceTraceEnums)
     spanKind = Azure::Core::Tracing::_internal::SpanKind::Server;
     int i = static_cast<int>(spanKind);
     i += 1;
+    (void)i;
```

Comment:
IIRC this was added because I had a compilation failure referencing an unreferenced variable.

Reply:
We should be hitting this in `main` too, then, correct? Do you recall why main builds fine? The variable is used, but not  after the increment, maybe the error is compiler/build environment dependent.

https://github.com/Azure/azure-sdk-for-cpp/pull/4608

If it isn't necessary, revert the changes to this file?

And actually, on a side-note, why do we have `i += 1` anyway? Maybe it's there to avoid an unreferenced variable error?

Reply:
Honestly, I can't remember the details - the test was written a year ago, and the error occurred weeks and weeks ago.

But line 23 mirrors line 31.